.claude/scripts/maintainer-preflight.test.sh

```
@@ -38,9 +38,37 @@ grep -Fq 'classify the saved login as indeterminate' "${run_loop}" ||
+  fail "GraphQL fallback does not preserve the failing probe's credential context"
+
+grep -Fq 'HTTP **401**' "${run_loop}" ||
+  fail "missing the explicit HTTP 401 authentication-rejection criterion"
+
+grep -Fq 'non-rate-limit' "${run_loop}" ||
+  fail "missing the non-rate-limit 403 credential-rejection criterion"
+
+grep -Fq 'A REST 5xx (or' "${run_loop}" ||
+  fail "missing the REST-503-plus-GraphQL-success regression rule"
+
+grep -Fq 'recommend' "${run_loop}" && grep -Fq 'gh auth login' "${run_loop}" ||
```

Shellcheck (0.11.0)

[info] Note that A && B || C is not if-then-else. C may run when A is true.

(SC2015)